filterLists_test.py

```diff
@@ -0,0 +1,161 @@
+from filterLists import *
+import os
+
+
+WHITE_TESTFILE = "whitelist.txt"
+BLACK_TESTFILE = "blacklist.txt"
+BUG_TESTFILE = "bug_report.txt"
+
+
+class testFilterBugReport():
+	def __init__(self):
+		print("Testing Filter Bug Report")
+
+	def removeTestFiles(self):
+		try:
+			os.remove(WHITE_TESTFILE)
+			os.remove(BLACK_TESTFILE)
+			os.remove(BUG_TESTFILE)
+		except Exception:
+			print()
+	def testInit(self):
+		"""
+        Test Case 1: Test files are not present, no files should be loaded or create during init
+		"""
+		print("Begin testInit")
+		self.removeTestFiles()
+		fil = filterBugReport()
+		try:
+			f1 = open(WHITE_TESTFILE,'r')
+			f2 = open(BLACK_TESTFILE,'r')
+			f3 = open(BUG_TESTFILE,'r')
+		except Exception:
+			print("Test Case 1 Pass")
+			print("End testInit")
+			return
+		print("Test Case 1 Fail")
+		raise Exception
+		print("End testInit")
+
+	def testCheckInBoth(self):
+		"""
+    Test Case 1: White and Black lists share subset, exception should be raised
+    Test Case 2: White and Black lists do not share a subset, no exception should be raised
```

Comment:
Read the [documentation](https://www.python.org/dev/peps/pep-0008/#documentation-strings) on strings.